mardown_converter.py

```
@@ -1,40 +1,82 @@
   ];
 
   let html = md;
-  for (let {regex, replace} of rules) {
+
+  // Apply rules in sequence
+  for (let { regex, replace } of rules) {
     html = html.replace(regex, replace);
   }
 
-  // list wrapping — only wraps first <li>
+  // Wrap unordered lists — only first match
```

Ruff (0.12.2)

SyntaxError: Expected a statement

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Got unexpected token —

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Expected an expression or a ')'

---

SyntaxError: Expected an expression

---

SyntaxError: Expected an expression

---

SyntaxError: Expected an identifier

---

SyntaxError: Expected an expression

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected ',', found '/'

---

SyntaxError: Expected an expression

---

SyntaxError: Expected an expression

---

SyntaxError: Expected a statement

---

SyntaxError: Expected a statement

---

Statement ends with an unnecessary semicolon

(E703)